configuration/config-txt/video.md

```diff
@@ -336,6 +336,38 @@ Note that this simply **creates** the mode (group 2 mode 87). In order to make t
 
 This may not work if your monitor does not support standard CVT timings.
 
+### LCD display/touchscreen options
+
+#### ignore_lcd
+
+By default no DSI LCD detection is run (`ignore_lcd=1`). To enable detection (and accordingly the LCD display), use `ignore_lcd=0`
+
+#### display_default_lcd
+
+Specify which display is to be used as the default. To set the LCD as the default, use `display_default_lcd=1`; to use HDMI as the default, use `display_default_lcd=0`
+
+#### lcd_framerate
+
+Specify the framerate of the Raspberry Pi LCD display, in Hertz/fps. Defaults to 60Hz.
+
+#### lcd_rotate
+
+Rotates the LCD output to match an inverted display. This will also rotate the touchscreen coordinates to match. Valid values are 0,1,2,3.
```

Comment:
No, this only supports flips and not transposes. It is a much cheaper operation than display_rotate as it uses the LCD configuration to enable the flips.